src/Task/useCases/getTaskByID/GetTaskByTaskIDUseCase.ts

```diff
@@ -0,0 +1,21 @@
+import { GetTaskByTaskIDResponse } from './GetTaskByTaskIDResponse';
+import { GetTaskByTaskIDRepoInterface } from '../../repos/getTask/IGetTaskByTaskIDRepository';
+
+export class GetTaskByTaskIDUseCase {
+  constructor(private readonly taskRepo: GetTaskByTaskIDRepoInterface) {}
+
+  async execute(taskId: string): Promise<GetTaskByTaskIDResponse> {
+    try {
+      const resultTask = await this.taskRepo.getTaskByTaskID(taskId);
+
+      return {
+        taskId: resultTask.taskId as string,
+        title: resultTask.title,
+        description: resultTask.description,
+        userid: resultTask.userid,
+      };
+    } catch (error) {
+      throw new Error((error as Error).message);
```

Comment:
I think we just need to throw exception to controller to catch the error,
 no need to use try/catch and throw error in catch

Reply:
Could you please check my latest commit for this fix?